### ISSUE TITLE
add tx simulation

### ISSUE THREAD

**Issue by adrienlacombe** [LedgerHQ/clear-signing-erc7730-registry#78]
ideally I would like to pass some tx ids and see what Flex would have displayed

**Comment by jnicoulaud-ledger** [comment]
We have a preview tool, it's a webapp you will have to launch locally for now: https://github.com/LedgerHQ/clear-signing-erc7730-registry/blob/master/developer-preview/README.md

**Comment by adrienlacombe** [comment]
> We have a preview tool, it's a webapp you will have to launch locally for now: https://github.com/LedgerHQ/clear-signing-erc7730-registry/blob/master/developer-preview/README.md

yes I did that but it doesn't allow me to put a tx and see what would have been screens, or did I miss smth?

**Comment by adrienlacombe** [comment]
I can't change

![image](https://github.com/user-attachments/assets/4e4d0719-c300-4e01-b25f-4b6148316e8b)

**Comment by adrienlacombe** [comment]
> Yes you didn't miss anything, it doesn't support it for now. It will in the future, either in this webapp or erc7730 CLI tool, but I don't know when

when it will happen, let me know here ;) thanks

**Comment by kuzdogan** [comment]
Check out https://playground.sourcify.dev/
